main/src/ca/uwaterloo/flix/language/phase/Eraser.scala

```diff
@@ -2,11 +2,12 @@ package ca.uwaterloo.flix.language.phase
 
 import ca.uwaterloo.flix.api.Flix
 import ca.uwaterloo.flix.language.CompilationError
-import ca.uwaterloo.flix.language.ast.ErasedAst.JType
 import ca.uwaterloo.flix.language.ast.ErasedAst.JType._
-import ca.uwaterloo.flix.language.ast.{ErasedAst, FinalAst}
+import ca.uwaterloo.flix.language.ast.ErasedAst.{ErasedType, JType}
+import ca.uwaterloo.flix.language.ast.{ErasedAst, FinalAst, MonoType}
 import ca.uwaterloo.flix.util.Validation
 import ca.uwaterloo.flix.util.Validation._
+import ca.uwaterloo.flix.language.ast.Symbol
 
 object Eraser extends Phase[FinalAst.Root, FinalAst.Root]  {
```

Comment:
Does the phase now run without crashing?

Reply:
yes, although not anymore